src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3125,6 +3168,38 @@ components:
           type: string
           example:
             "Main group"
+    GetClearingHistory:
+      type: object
+      properties:
+        date:
+          type: string
+          format: date-time
+          description: Date of the clearing history
+          example: "2023-06-15"
+        username:
+          type: string
+          description: Username of the user who cleared the package
+          example: "fossy"
+        scope:
+          type: string
+          description: Scope of the clearing
+          example: "local"
```

Comment:
Can add enum with "local", "package" and "global" values. (as per `\Fossology\Lib\Data\DecisionScopes::__construct()`)